submissions/tests/test_api.py

```diff
@@ -164,13 +164,46 @@ def test_get_submission(self):
         with self.assertRaises(api.SubmissionNotFoundError):
             api.get_submission("deadbeef-1234-5678-9100-1234deadbeef")
 
-    @patch.object(Submission.objects, 'get')
+    @mock.patch.object(Submission.objects, 'get')
     @raises(api.SubmissionInternalError)
     def test_get_submission_deep_error(self, mock_get):
         # Test deep explosions are wrapped
         mock_get.side_effect = DatabaseError("Kaboom!")
         api.get_submission("000000000000000")
 
+    def test_get_old_submission(self):
+        # hack in an old-style submission, this can't be created with the ORM (EDUCATOR-1090)
+        with transaction.atomic():
+            student_item = StudentItem.objects.create()
+            connection.cursor().execute("""
+                INSERT INTO submissions_submission
+                    (id, uuid, attempt_number, submitted_at, created_at, raw_answer, student_item_id, status)
+                VALUES (
+                    {}, {}, {}, {}, {}, {}, {}, {}
+                );""".format(
+                    1,
+                    "\'deadbeef-1234-5678-9100-1234deadbeef\'",
+                    1,
+                    "\'2017-07-13 17:56:02.656129\'",
+                    "\'2017-07-13 17:56:02.656129\'",
+                    "\'{\"parts\":[{\"text\":\"raw answer text\"}]}\'",
+                    int(student_item.id),
+                    "\'A\'"
+                ), []
+            )
+
+        with mock.patch.object(
+            Submission.objects, 'raw',
+            wraps=Submission.objects.raw
+        ) as mock_raw:
+            _ = api.get_submission('deadbeef-1234-5678-9100-1234deadbeef')
+            self.assertEqual(1, mock_raw.call_count)
+
+            # On subsequent accesses we still get the submission, but raw() isn't needed
```

Comment:
Then should this block not be indented within the mock_raw patch object (since raw() isn't needed)?